maestro-utils/src/main/kotlin/InsightsInterface.kt

```diff
@@ -0,0 +1,38 @@
+package maestro.utils
+
+interface InsightsInterface {
```

Comment:
nit: interfaces shouldn't include "Interface" on their name.

Reply:
Good point, I'll fix this

Reply:
Done ✅